lacework/resource_lacework_policy_compliance.go

```diff
@@ -0,0 +1,255 @@
+package lacework
+
+import (
+	"errors"
+	"fmt"
+	"log"
+	"strings"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/lacework/go-sdk/api"
+)
+
+const policyType = "Compliance"
```

Comment:
I love that you declared a constant here! Great job.

Though, we already expose one constant in the go-sdk, could we use it instead?

https://github.com/lacework/go-sdk/blob/main/api/policy.go#L56

You should be able to use it like `api.PolicyTypeCompliance.String()` below